SKILL.tsv

```diff
@@ -1085,116 +1085,116 @@ SKILL_20150317_001084	Attack #{SkillAtkAdd}#{nl}3 Magic Missiles
 SKILL_20150317_001085	Quick Cast
 SKILL_20150317_001086	Cast Time decreased.
 SKILL_20150317_001087	Cast Time - #{CaptionRatio}#% {nl} Duration #{CaptionTime}# seconds
-SKILL_20150317_001088	Fire Ball
-SKILL_20150317_001089	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Summons a stationary fireball. You can blow it up by hitting it with a sword attack.
+SKILL_20150317_001088	$Fire Ball
+SKILL_20150317_001089	${#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Summons a stationary fireball. You can take it out by hitting it with a sword attack.
```

Comment:
As I remember, when you hit a fireball you "push it forward", or "launch it forward".
"Take it out" sounds as if you are getting the fire from your pocket.